<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>838</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MERGE</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>D:/yingongsui/Pictures/icon.png</normaloff>D:/yingongsui/Pictures/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_10">
    <item row="5" column="0" rowspan="7" colspan="2">
     <widget class="MyTextViewer" name="MyTBrowser">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei</family>
       </font>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="placeholderText">
       <string>Please choose files or drag files here</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="MyTextEdit" name="textEdit">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei</family>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="placeholderText">
       <string>input the path of ffmpeg.exe</string>
      </property>
     </widget>
    </item>
    <item row="10" column="3">
     <widget class="QComboBox" name="exacomboBox">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei UI</family>
       </font>
      </property>
      <item>
       <property name="text">
        <string>.aac</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>.mp4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>.mp3</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="File1Button">
      <property name="minimumSize">
       <size>
        <width>125</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei</family>
       </font>
      </property>
      <property name="text">
       <string>Choose Files</string>
      </property>
     </widget>
    </item>
    <item row="11" column="2">
     <widget class="QPushButton" name="avmButton">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei</family>
       </font>
      </property>
      <property name="text">
       <string>A+V Merge</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QLCDNumber" name="testlcdNumber"/>
    </item>
    <item row="9" column="3">
     <widget class="QComboBox" name="typecomboBox">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei UI</family>
       </font>
      </property>
      <property name="placeholderText">
       <string>Type</string>
      </property>
      <item>
       <property name="text">
        <string>.mp4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>.avi</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="2" colspan="3">
     <widget class="QPushButton" name="ffmpegButton">
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei</family>
       </font>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item row="9" column="2">
     <widget class="QPushButton" name="mergeButton">
      <property name="minimumSize">
       <size>
        <width>125</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>125</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Merge/Transcode</string>
      </property>
     </widget>
    </item>
    <item row="8" column="2" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLCDNumber" name="lcdNumber">
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>30</pointsize>
         </font>
        </property>
        <property name="text">
         <string>S</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="7" column="3">
     <widget class="QPushButton" name="testButton">
      <property name="text">
       <string>Test</string>
      </property>
     </widget>
    </item>
    <item row="15" column="0" colspan="4">
     <widget class="QProgressBar" name="progressBar">
      <property name="font">
       <font>
        <family>Yu Gothic</family>
       </font>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="10" column="2">
     <widget class="QPushButton" name="extractButton">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei</family>
       </font>
      </property>
      <property name="text">
       <string>Extract</string>
      </property>
     </widget>
    </item>
    <item row="16" column="3">
     <widget class="QPushButton" name="clearButton">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>30</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Clear All</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="4" alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>70</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei UI</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Video and Text Files Merger</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QTextBrowser" name="textBrowser">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>25</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color : rgba(255,255,255,0%)</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;ffmpeg link: https://www.gyan.dev/ffmpeg/builds/packages/ffmpeg-2022-10-27-git-00b03331a0-full_build.7z&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>838</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyTextEdit</class>
   <extends>QTextEdit</extends>
   <header>MyOwnWidgets</header>
  </customwidget>
  <customwidget>
   <class>MyTextViewer</class>
   <extends>QTextBrowser</extends>
   <header>MyOwnWidgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
